ambry-api/src/main/java/com/github/ambry/config/ClusterMapConfig.java

```diff
@@ -320,5 +324,7 @@ public ClusterMapConfig(VerifiableProperties verifiableProperties) {
         new JSONObject(verifiableProperties.getString("clustermap.recovery.test.hardware.layout", "{}"));
     clustermapRecoveryTestPartitionLayout =
         new JSONObject(verifiableProperties.getString("clustermap.recovery.test.partition.layout", "{}"));
+    clustermapMaxDiskCapacityInBytes =
+        verifiableProperties.getLong(CLUSTERMAP_MAX_DISK_CAPACITY_IN_BYTES, 10L * 1024 * 1024 * 1024 * 1024);
```

Comment:
Could you change this to `getLongInRange`?  The parameters are:  min = 0, max = Long.MAX_VALUE, default = 20L * 1024 * 1024 * 1024 * 1024 (let's use 20TB as default value)

Reply:
done.